### DISCUSSION TITLE
RecordRTC  chunks throwing Bad Request: Invalid data received.

### DISCUSSION THREAD

**Discussion by singhalpk** [deepgram/community#204] [General help] [answered]
Hi Team,
what's the required configuration for  raw binary audio data like I tried for audio/wav base 64 but it's throwing error
btw i am using RecordRTC configuration is 
```
 type: 'audio',
  mimeType: 'audio/webm', 
  recorderType: StereoAudioRecorder,
  timeSlice: 10000, 
  desiredSampRate: 16000,
```
  Any specific params requirements ?

**Comment by nikolawhallon** [comment, marked as answer]
We have some docs which might be useful: https://developers.deepgram.com/docs/determining-your-audio-format-for-live-streaming-audio

It looks like you may need to attach `sample_rate=16000&channels=2&multichannel=true&encoding=linear16` or `sample_rate=16000&channels=2&multichannel=true&encoding=opus` if the audio is raw (it's not clear to me if RecordRTC uses PCM or Opus, looking at https://recordrtc.org/ I think it could be either).

However, I think `webm` audio is containerized, so these parameters maybe need not be specified. Can you share the full URL/query parameters you use for your requests, and maybe a request ID for a failing request?
